### PR TITLE
remove content hub onboarding

### PR DESCRIPTION
**Pull request by proevan**

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/utils/DialogUtils.kt [hunk 1]
```diff
@@ -313,40 +313,6 @@ object DialogUtils {
...
 
-    fun showContentServiceOnboardingSpotlight(
-        activity: FragmentActivity,
-        targetView: View,
-        dismissListener: DialogInterface.OnDismissListener,
-        ok: View.OnClickListener?
-    ): Dialog = SpotlightDialog.Builder(activity, targetView)
-            .spotlightConfigs(
-                RectangleSpotlightConfigs(
-                    width = activity.resources.getDimensionPixelSize(R.dimen.content_service_focus_view_width),
```
Comment (benjamin-cheng) [inline app/src/main/java/org/mozilla/focus/utils/DialogUtils.kt:324]: Could we also remove the `content_service_focus_view_width` and `content_service_focus_view_radius` definitions from `dimens.xml`?

File: app/src/main/res/layout/onboarding_spotlight_content_services_logo_man.xml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:src="@drawable/logo_man_copy_3"/>
```
Comment (benjamin-cheng) [inline app/src/main/res/layout/onboarding_spotlight_content_services_logo_man.xml:5]: Could we remove the `logo_man_copy_3` asset?

File: app/src/main/java/org/mozilla/focus/utils/DialogUtils.kt (hunk 1 above)
Comment (proevan) [reply app/src/main/java/org/mozilla/focus/utils/DialogUtils.kt:324]: those two assets are still used in DialogUtils.showContentServiceRequestClickSpotlight

File: app/src/main/res/layout/onboarding_spotlight_content_services_logo_man.xml (hunk 2 above)
Comment (proevan) [reply app/src/main/res/layout/onboarding_spotlight_content_services_logo_man.xml:5]: fixed in the last commit, thanks for mention

**Pull request merged**